### PR TITLE
Write a basic ResultList ontology

### LINKED ISSUE #131: Create API ontology for ResultList and Error
We need to define types that only exist in the API.

Initially these are ResultSet (for a list response) and Error (for error responses).

### PR DESCRIPTION
**Pull request by alexwlchan**
Resolves #131.

### CODE REVIEW

File: ontologies/Schema/resultlist.ttl [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+<http://wellcomecollection.org/ontologies/resultlist>
+    rdf:type owl:Ontology ;
+    dcterms:title "Result list ontology"@en ;
+    dcterms:created "2017-11-07"^^xsd:date ;
+    dcterms:description "Ontology describing the properties of a result list"@en .
+
+wresultlist:ResultList rdf:type owl:Class ;
+    rdfs:label "Result list"@en ;
+    rdfs:comment "A collection of works."@en ;
```
Comment (jamesgorrie) [inline ontologies/Schema/resultlist.ttl]: Only used for works?
Why not a WorksList then?

Comment (silveroliver) [reply ontologies/Schema/resultlist.ttl]: Its a good point. Would we use result lists for lists of items, holdings and potentially concepts in the future. @jamesgorrie I will pick up with platform team tomorrow.

File: ontologies/Schema/resultlist-ontology.ttl [hunk 2]
```diff
@@ -15,13 +14,18 @@
...
 wresultlist:ResultList rdf:type owl:Class ;
     rdfs:label "Result list"@en ;
-    rdfs:comment "A collection of works."@en ;
+    rdfs:comment "A collection of things."@en ;
     rdf:isDefinedBy <http://wellcomecollection.org/ontologies/resultlist> .
 
+wresultlist:Thing rdf:type owl:Class ;
+    rdfs:label "Thing"@en ;
+    rdfs:comment "A thing"@en ;
+    rdf:isDefinedBy <http://wellcomecollection.org/ontologies/wresultlist:> .
```
Comment (jtweed) [inline ontologies/Schema/resultlist-ontology.ttl]: Stray colon.

Comment (alexwlchan) [reply ontologies/Schema/resultlist-ontology.ttl]: Thanks, fixed.

**Comment by alexwlchan** [comment]
I’ve discussed this with @silveroliver; we’re both happy with this as-is.

**Pull request merged**